res/poketool/pokegra/pl_pokegra/meson.build

```diff
@@ -0,0 +1,92 @@
+pl_pokegra_srcs = []
+pl_pokegra_files = []
+
+sprites_flags = ['-scanfronttoback', '-handleempty']
+palette_flags = ['-bitdepth', '8', '-nopad', '-comp', '10']
+
+foreach i : range(494)
+    padded_i = run_command('printf', '%04d', i.to_string(), check: false).stdout()
+
+    female_back = files(padded_i / 'female' / 'back.png')
+    male_back = files(padded_i / 'male' / 'back.png')
+    female_front = files(padded_i / 'female' / 'front.png')
+    male_front = files(padded_i / 'male' / 'front.png')
+    pl_pokegra_srcs += [female_back, male_back, female_front, male_front]
+
+    pl_pokegra_files += custom_target(padded_i + '-00.NCGR',
+        output: padded_i + '-00.NCGR',
+        input: female_back,
+        command: [
+            nitrogfx_exe,
+            '@INPUT@',
+            '@OUTPUT@',
+            sprites_flags
+        ]
+    )
+    pl_pokegra_files += custom_target(padded_i + '-01.NCGR',
+        output: padded_i + '-01.NCGR',
+        input: male_back,
+        command: [
+            nitrogfx_exe,
+            '@INPUT@',
+            '@OUTPUT@',
+            sprites_flags
+        ]
+    )
+    pl_pokegra_files += custom_target(padded_i + '-02.NCGR',
+        output: padded_i + '-02.NCGR',
+        input: female_front,
+        command: [
+            nitrogfx_exe,
+            '@INPUT@',
+            '@OUTPUT@',
+            sprites_flags
+        ]
+    )
+    pl_pokegra_files += custom_target(padded_i + '-03.NCGR',
+        output: padded_i + '-03.NCGR',
+        input: male_front,
+        command: [
+            nitrogfx_exe,
+            '@INPUT@',
+            '@OUTPUT@',
+            sprites_flags
+        ]
+    )
+
+    if i == 0
+        # nitrogfx doesn't like these two files, so copy over the canonical versions
```

Comment:
Research pending, of course, but I wonder if the original tools would compress/format these properly.

Reply:
I imagine so. `nitrogfx` gets close, but it replaces the long pairs of `FF FF` at the end of the palette file with `FF 7F`. I'm not quite sure why, and I'm not familiar enough with `nitrogfx` myself to debug it.

Reply:
>I'm not quite sure why
In an RGB555 palette, when stored in 16 bits, the highest bit is unused. I figure nitrogfx used it to signify a missing color maybe?

Reply:
It's probably a missing color, then, yeah. Is there a way we could modify nitrogfx to support it? Maybe a new input flag?